config/manifests/bases/aws-load-balancer-operator.clusterserviceversion.yaml

```diff
@@ -6,6 +6,7 @@ metadata:
     capabilities: Basic Install
     olm.skipRange: <0.2.0
     operatorframework.io/suggested-namespace: aws-load-balancer-operator
+    operators.openshift.io/infrastructure-features: '["proxy-aware"]'
```

Comment:
Could you help me understand what this does?

Reply:
It's a tag which informs the user of the operator that this operator is proxy aware ([doc](https://docs.openshift.com/container-platform/4.12/operators/operator_sdk/osdk-generating-csvs.html#osdk-csv-manual-annotations_osdk-generating-csvs)). 

The user will be able to filter operators by the presence of this feature:
![image](https://user-images.githubusercontent.com/18031474/216006832-d2d6c439-c962-4ac4-8c4e-273ad55822f1.png)

Reply:
Oh I see, that's neat!